### PR TITLE
[ClusterLoader2] Enable custom timeouts for probes

### PR DESCRIPTION
**Pull request by jprzychodzen**
Allow probes measurement to use custom timeout for pod readiness. Required to adapt containerd for tests

Ref. #1110 

/assign @mborsz

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/probes/probes.go [hunk 1]
```diff
@@ -37,8 +37,8 @@ const (
 
 	manifestsPathPrefix = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/measurement/common/probes/manifests/"
 
-	checkProbesReadyInterval = 15 * time.Second
-	checkProbesReadyTimeout  = 5 * time.Minute
+	defaultCheckProbesReadyInterval = 15 * time.Second
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/probes/probes.go:40]: Let's not expose the interval as a var - timeout seems to be enough for now.

Comment (jprzychodzen) [reply clusterloader2/pkg/measurement/common/probes/probes.go]: Make sense. We don't need it and we can easily expose it if needed.

**Pull request merged**